editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+                [
+                    {
+
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 1",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 1",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669dfff22f8a5b002ce60115?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Dubai",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Summer, Fashion"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 2",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 2",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e00122f8a5b002ce60121?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Cosmoso",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Architecture"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 3",
```

Comment:
use singular and non-contain number names.
```suggestion
                        displayer: "Card",
```